sklearn/utils/_repr_html/params.py

```diff
@@ -50,12 +129,26 @@ def _params_html_repr(params):
             ></i></td>
             <td class="param">{param_name}&nbsp;</td>
             <td class="value">{param_value}</td>
+            <td class="doc_link"><a class="sk-estimator-doc-link
+                                 {doc_link}
+                                </a>
+            </td>
```

Comment:
The `?` link is indeed quite small. I wonder if we could make the whole cell containing the `?` clickable to be more user friendly. It seems to be a `<td class="doc_link">`